### PR TITLE
Changed all the links associated with getting started 

### PR DESCRIPTION
**Pull request by gabzillah**
Hope this helps! Please let me know if you would prefer a different link or format! Thanks! 😸

### CODE REVIEW

File: tests/dummy/app/templates/modules/accordion.hbs [hunk 1]
```diff
@@ -13,7 +13,7 @@
       <div class="ui hidden divider"></div>
 
       <p>
-        For additional documentation on how to configure this module, please refer to the <a href="/">Getting Started</a> section and Semantic UI's module specific external documentation links below.
+        For additional documentation on how to configure this module, please refer to the <a href="http://semantic-org.github.io/Semantic-UI-Ember/#/modules">Getting Started</a> section and Semantic UI's module specific external documentation links below.
```
Comment (aaronbhansen) [inline tests/dummy/app/templates/modules/accordion.hbs]: Thanks for this! I didn't realize I missed that. 
One small change, lets use link-to to ensure if the documentation changes in the future the urls work. Just change all the urls to `{{#link-to 'modules.index'}}Getting Started{{/link-to}}`

Comment (gabzillah) [inline tests/dummy/app/templates/modules/accordion.hbs]: Sure no problem!

**Comment by aaronbhansen** [comment]
One other thing, can you collapse all those commits down to one when you're finished. I usually don't mind extra commits, I think it would be easier to read as just one.

**Comment by aaronbhansen** [comment]
Great, thank you!

**Pull request merged**